### ISSUE TITLE
Docker image tiledb-presto:latest (1.3.0) doesn't work

### ISSUE THREAD

**Issue by mnordie** [TileDB-Inc/TileDB-Presto#34]
The readme says `docker run -it --rm tiledb/tiledb-presto`, but the image never stop initializing:

```
docker run -it --rm tiledb/tiledb-presto
Not running
Started as 13
Waiting for presto to initialize............
```

Checking /opt/presto-server-310/data/var/log/server.log:
```
2019-06-18T13:01:25.820Z        INFO    main    io.prestosql.server.PluginManager       -- Loading plugin /opt/presto-server-310/data/plugin/tiledb --
2019-06-18T13:01:25.820Z        DEBUG   main    io.prestosql.server.PluginManager       Classpath for tiledb:
2019-06-18T13:01:25.820Z        DEBUG   main    io.prestosql.server.PluginManager           /opt/presto-server-310/data/plugin/tiledb/presto-tiledb-1.3.0.jar
2019-06-18T13:01:25.824Z        ERROR   main    io.prestosql.server.PrestoServer        No service providers of type io.prestosql.spi.Plugin
java.lang.IllegalStateException: No service providers of type io.prestosql.spi.Plugin
        at com.google.common.base.Preconditions.checkState(Preconditions.java:588)
        at io.prestosql.server.PluginManager.loadPlugin(PluginManager.java:166)
        at io.prestosql.server.PluginManager.loadPlugin(PluginManager.java:157)
        at io.prestosql.server.PluginManager.loadPlugins(PluginManager.java:138)
        at io.prestosql.server.PrestoServer.run(PrestoServer.java:125)
        at io.prestosql.server.PrestoServer.main(PrestoServer.java:70)
```

This is on windows 10 with Docker desktop 2.0.5.0

**Comment by Shelnutt2** [comment]
@mnordie thank you for reporting this. We've merged a fix, and I tagged a new release, [1.3.1](https://github.com/TileDB-Inc/TileDB-Presto/releases/tag/1.3.1). The latest docker image is updated and should now be functional. We also include a few other minor improvements in 1.3.1 including updating to the latest prestosql version of 315.

**Comment by mnordie** [comment]
@Shelnutt2 Awesome response time, 1.3.1 works!
